security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -12,6 +12,24 @@ These release notes track the development of {cert-manager-operator}.
 
 For more information, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
 
+[id="cert-manager-operator-release-notes-1-14-1"]
+== {cert-manager-operator} 1.14.1
+
+Issued: 2024-10-17
```

Comment:
Planned release date is 04th November, 2024 so ti's can be updated accordingly.

Reply:
Ack